x-pack/plugins/search_indices/public/components/quick_stats/quick_stats.tsx

```diff
@@ -107,6 +108,10 @@ export const QuickStats: React.FC<QuickStatsProps> = ({ index, mappings, indexDo
                 description: index.size ?? '0b',
               },
             ]}
+            tooltipContent={i18n.translate('xpack.searchIndices.quickStats.documentCountTooltip', {
+              defaultMessage:
+                'This excludes nested documents, which Elasticsearch uses internally to store chunks of vectors.',
```

Comment:
```suggestion
                'This excludes nested documents, which semantic_text uses internally to store chunks of vectors.',
```

Reply:
I wanted to make this wider than just because we also use that for more manual vector ingestion.